qlty-check/src/parser/ripgrep.rs

```diff
@@ -100,9 +100,9 @@ impl Parser for Ripgrep {
                             path: path.text.clone(),
                             range: Some(Range {
                                 start_line: line_number,
-                                start_column: submatch.start,
+                                start_column: submatch.start + 1,
```

Comment:
Not sure if there's a docs page which references this behavior but  if not could you please add a comment just clarifying the discovery and the need for this +1?